modules/account-lib/src/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,55 @@
+import { bigIntFromBufferBE, bigIntToBufferBE } from '../util';
+import BaseCurve from './baseCurve';
+import * as secp from '@noble/secp256k1';
+
+const order = secp.CURVE.n;
+
+export class Secp256k1Curve implements BaseCurve {
+  scalarRandom(): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.randomPrivateKey()));
+  }
+
+  scalarAdd(x: bigint, y: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateAdd(x, y.toString(16))));
+  }
+
+  scalarSub(x: bigint, y: bigint): bigint {
+    const negatedY = Buffer.from(secp.utils.privateNegate(y)).toString('hex');
```

Comment:
Should be able to pass `secp.utils.privateNegate(y)` into the next line without converting, I think (because Buffer satisfies Uint8Array).